skt/executable.py

```diff
@@ -558,39 +560,35 @@ def load_config(args):
 
     if config.has_section('config'):
         for (name, value) in config.items('config'):
-            if name not in cfg or cfg.get(name) is None:
+            if not cfg.get(name):
                 cfg[name] = value
 
-    if config.has_section('publisher') and ('publisher' not in cfg or
-                                            cfg.get('publisher') is None):
+    if config.has_section('publisher') and not cfg.get('publisher'):
```

Comment:
Hmm, we'll need better name differentiation than `cfg` vs `config` here.

Reply:
Agreed, a lot of naming is confusing.